redis/scan.go

```diff
@@ -134,6 +150,8 @@ func convertAssignValue(d reflect.Value, s interface{}) (err error) {
 		err = convertAssignBulkString(d, s)
 	case int64:
 		err = convertAssignInt(d, s)
+	case float64:
```

Comment:
Explain why this is needed.  

Redigo does not use floats to represent Redis values.

Reply:
I need to store float lists in Redis so I need to get them back somehow to golang client.

Reply:
The Redis result types are error, simple string, bulk string, integer and array.  Redigo does not use float to represent any of these types.